models/pull.go

```diff
@@ -517,7 +518,7 @@ func (pr *PullRequest) apiFormat(e Engine) *api.PullRequest {
 	}
 
 	if pr.Status != PullRequestStatusChecking {
-		mergeable := pr.Status != PullRequestStatusConflict && !pr.IsWorkInProgress()
+		mergeable := (pr.Status != PullRequestStatusConflict || pr.Status != PullRequestStatusError) && !pr.IsWorkInProgress()
```

Comment:
https://play.golang.org/p/g7lTrhTLrs3
```suggestion
		mergeable := !(pr.Status == PullRequestStatusConflict || pr.Status == PullRequestStatusError) && !pr.IsWorkInProgress()
```

Reply:
as I've said before I'm an idiot